src/main/java/org/folio/circulation/domain/validation/RequestQueueValidation.java

```diff
@@ -87,24 +87,16 @@ private RequestQueueValidation() {
     }
 
     return result.value().isQueueForInstance()
-      ? pageRequestsHaveTopPositions(result)
-      : pageRequestHasFirstPosition(result);
+    ? result
+    : pageRequestHasFirstPosition(result);
   }
 
   /**
-   * Makes sure that all Page requests are always at the top of the unified queue (TLR feature
-   * enabled).
+   * Makes sure that all Page requests are always at the first position of the unified queue.
```

Comment:
Remove this comment. This method is not for unified queue, it's for old item-based request queues.

Reply:
Comment is moved, thanks.